torch/testing/_creation.py

```diff
@@ -0,0 +1,159 @@
+"""
+This package contains tensor creation utilities.
+"""
+
+import torch
+from typing import Optional, Tuple, Union
+import math
+
+__all__ = [
+    "make_tensor"
+]
+
+def make_tensor(
+    size: Tuple[int, ...],
+    *,
+    device: Union[str, torch.device] = "cpu",
+    dtype: torch.dtype = torch.float32,
+    low: Optional[float] = None,
+    high: Optional[float] = None,
+    requires_grad: bool = False,
+    noncontiguous: bool = False,
+    exclude_zero: bool = False
+) -> torch.Tensor:
+    r"""Creates a random tensor with the given :attr:`size`.
+
+    The function comes with other optional arguments to allow random tensor generation for the user's needs.
+
+    By default, :attr:`device` is ``cpu`` and :attr:`dtype` is ``torch.float32``, but user can pass these arguments
+    as required.
+
+    If :attr:`low` and :attr:`high` are not passed, following default values are considered depending on the given
+    :attr:`dtype`:
+
+    - boolean type: `low` = 0, `high` = 2.
+    - uint8 type: `low` = 0, `high` = 9.
+    - floating and integral types: `low` = -9 and `high` = 9.
+    - complex types, for each real and imaginary part: `low` = -9, `high` = 9.
+
+    If :attr:`low` and :attr:`high` are passed, they are considered only if they are within the
```

Comment:
I would inline this description into the description of the first low or high kwarg and simplify it to: 

"If low or high are outside the range of the datatype's representable finite values then they are clamped to the lowest or highest representable finite value, respectively."